src/App.tsx

```diff
@@ -15,36 +16,84 @@
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  alphabet = 'alphabet',
+  length = 'length',
+}
+
+export const App = () => {
+  const [sortType, setSortType] = useState('');
+  const [reversed, setReversed] = useState(false);
+
+  function sortGoodsList(goods: string[], sortBy: string, isReverse: boolean) {
+    const sortedList = [...goods].sort((good1, good2) => {
+      switch (sortBy) {
+        case SortType.alphabet:
+          return good1.localeCompare(good2);
+        case SortType.length:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+
+    return isReverse ? sortedList.reverse() : sortedList;
+  }
+
+  const handleReset = () => {
```

Comment:
[CODE STYLE] - Function's name should start with a verb to clearly indicate what the method does. Here, `handleReset` could be renamed to `resetSortAndOrder` or similar to better describe its action.